models/issues/issue_project.go

```diff
@@ -123,25 +117,51 @@ func addUpdateIssueProject(ctx context.Context, issue *Issue, doer *user_model.U
 		}
 	}
 
-	if _, err := db.GetEngine(ctx).Where("project_issue.issue_id=?", issue.ID).Delete(&project_model.ProjectIssue{}); err != nil {
-		return err
+	if action == "null" {
```

Comment:
I think it's better to create different methods.

Reply:
Can you give an example?

Reply:
I mean using mutiple methods instead of one method but a parameter `action` for different logic.